apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -856,6 +856,12 @@ function handleUncaughtErrors( calypsoPort ) {
 
 async function handleEditorLoaded( calypsoPort ) {
 	await isEditorReadyWithBlocks();
+
+	const parentPageId = getQueryArg( window.location.href, 'parent_post' );
```

Comment:
should we name everything `parentPost` vs `parentPage`, at the end everything is a post.